tracer/test/snapshots/Iast.StackTraceLeak.AspNetCore2.verified.txt

```diff
@@ -0,0 +1,71 @@
+﻿[
+  {
+    TraceId: Id_1,
+    SpanId: Id_2,
+    Name: aspnet_core.request,
+    Resource: GET /iast/stacktraceleak,
+    Service: Samples.Security.AspNetCore2,
+    Type: web,
+    Error: 1,
+    Tags: {
+      aspnet_core.route: iast/stacktraceleak,
+      component: aspnet_core,
+      env: integration_tests,
+      error.msg: Custom exception message,
+      error.stack:
+System.SystemException: Custom exception message
+at Samples.Security.AspNetCore5.Controllers.IastController.StackTraceLeak(),
+      error.type: System.SystemException,
+      http.method: GET,
+      http.request.headers.host: localhost:00000,
+      http.route: iast/stacktraceleak,
+      http.status_code: 500,
+      http.url: http://localhost:00000/Iast/StackTraceLeak,
+      http.useragent: Mistake Not...,
+      language: dotnet,
+      runtime-id: Guid_1,
+      span.kind: server,
+      _dd.git.commit.sha: aaaaaaaaaaaaaaaaaaaaabbbbbbbbbbbbbbbbbbbbbbb,
+      _dd.git.repository_url: https://github.com/DataDog/dd-trace-dotnet,
+      _dd.iast.enabled: 1,
+      _dd.iast.json:
+{
+  "vulnerabilities": [
+    {
+      "type": "STACKTRACE_LEAK",
+      "evidence": {
+        "value": "Samples.Security.AspNetCore2,SystemException"
```

Comment:
Should this not be `System.Exception`?

Reply:
In the IastController, we are launching a System.SystemException :)